lib/actions/index.coffee

```diff
@@ -14,6 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ###
 
+
+# Use fast-boot to cache require lookups, speeding up startup
+require('fast-boot2').start({
+	cacheFile: '.fast-boot.json'
```

Comment:
Is this added in the directory where the cli is executed? I feel like it should be in the install location, or the home directory or something.

Reply:
Changed it to use __dirname to be consistently in the same place as the binary